projects/fsastorefrontlib/src/assets/translations/overrides/en/common.ts

```diff
@@ -57,6 +57,8 @@ export const common = {
   formErrors: {
     pattern: 'Should contain at least one number, no special characters',
     maxlength: 'Maximum {{ maxLength }} characters',
+    min: 'Minimum value is {{ minValue }} {{ measure }}',
+    max: 'Maximum value is {{maxValue }} {{ measure }}',
```

Comment:
formatting :)

Reply:
But I don't see that Travis is failed. As I can see there is nothing to format.